packages/opencensus-core/test/test-recorder.ts

```diff
@@ -176,6 +176,41 @@ describe('Recorder', () => {
           }
         });
 
+    describe(`for distribution aggregation data with attachments`, () => {
```

Comment:
Nit: I tend to only use template backticks when something needs to be interpolated

Reply:
This is due to copy-paste from previous tests, fixed.